app/Helpers/TerrafundDashboardQueryHelper.php

```diff
@@ -2,11 +2,17 @@
 
 namespace App\Helpers;
 
+use App\Models\V2\Projects\Project;
+
 class TerrafundDashboardQueryHelper
 {
-    public static function buildQueryFromRequest($query, $request)
+    public static function buildQueryFromRequest($request)
     {
+        $query = Project::query();
         $query = $query->where('framework_key', 'terrafund');
+        $query = $query->whereHas('organisation', function ($query) {
```

Comment:
A minor point, but there's no any real reason to assign the `$query` variable at each step
```
$query = Project::where('framework_key', 'terrafund')
    -> $whereHas('organisation', function($query) {
    ```